plyer/platforms/win/libs/screenmetric.py

```diff
@@ -0,0 +1,10 @@
+__all__ = ('get_SystemMetrics')
```

Comment:
If all this does is call the function from the other module, then no, there isn't any reason to have this module.